science/rtl_433/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 PortGroup           github 1.0
 
-github.setup        merbanan rtl_433 bf4f9b4ebd4da642dfc788052d1a99d5812519f8
-version             20160420
+github.setup        merbanan rtl_433 18.05
+version             18.05
```

Comment:
This may require an "epoch" bump (i.e. `epoch 1` to be added on a line after version, see the guide for why.)